python/chronos/src/bigdl/chronos/forecaster/__init__.py

```diff
@@ -64,16 +65,17 @@ def __exit__(self, *args, **opts):
     warnings.warn("Please install `bigdl-orca` to use full collection of forecasters.")
 
 # import forecasters
+PREFIX_PATH = 'bigdl.chronos.forecaster'
 if torch_available:
```

Comment:
check `torch_available` and many other variables by `importlib.util.find_spec`

Reply:
Except for `tensorflow` and `prophet`, other modules have been replaced.
Reason
* We will use `tf.__version__` to check if the user has the correct version installed.
* `Prophet` will print redundant information after using `importlib.util.find_spec`, like this (import `plotly` failure!), We don't need this package.